clusterloader2/pkg/measurement/util/prometheus.go

```diff
@@ -174,8 +174,13 @@ func (qr *promResponseData) UnmarshalJSON(b []byte) error {
 
 // ToPrometheusTime returns prometheus string representation of given time.
 func ToPrometheusTime(t time.Duration) string {
-	if t < time.Minute {
-		return fmt.Sprintf("%ds", int64(t)/int64(time.Second))
+	minutes := int64(t) / int64(time.Minute)
+	seconds := (int64(t) % int64(time.Minute)) / int64(time.Second)
+	if minutes == 0 {
+		return fmt.Sprintf("%ds", seconds)
 	}
-	return fmt.Sprintf("%dm", int64(t)/int64(time.Minute))
+	if seconds == 0 {
+		return fmt.Sprintf("%dm", minutes)
+	}
+	return fmt.Sprintf("%dm%ds", minutes, seconds)
```

Comment:
Can we just produce output in seconds instead of doing some calculations here?

Like 1012312312s?

Reply:
Done